source/extensions/filters/http/cache/cache_filter_logging_info.h

```diff
@@ -0,0 +1,116 @@
+#pragma once
+
+#include "envoy/stream_info/filter_state.h"
+
+#include "absl/strings/str_format.h"
+
+namespace Envoy {
+namespace Extensions {
+namespace HttpFilters {
+namespace Cache {
+
+enum class CacheLookupStatus {
```

Comment:
nit: You don't need a 'Cache' prefix on the enum class name because its in namespace 'Cache' already, but either way is OK with me.